_data/simple-icons.json

```
@@ -7282,6 +7282,11 @@
             "hex": "DF0522",
             "source": "https://uk.kyocera.com"
         },
+        {
+            "title": "L'Équipe",
+            "hex": "D61E00",
```

```suggestion
            "hex": "E42829",
```
From the source logo, this was the color I saw.